monai/networks/trt_compiler.py

```
@@ -85,11 +85,22 @@ def cuassert(cuda_ret):
+        RuntimeError: If the CUDA call returned an error.
     """
     err = cuda_ret[0]
-    if err != 0:
-        raise RuntimeError(f"CUDA ERROR: {err}")
+    if err.value != 0:
+        err_msg = f"CUDA ERROR: {err.value}"
+        try:
+            _, err_name = cudart.cudaGetErrorName(err)
+            _, err_str = cudart.cudaGetErrorString(err)
+            err_msg = f"CUDA ERROR {err.value}: {err_name} — {err_str}"
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)